internal/kokoro/vet.sh

```diff
@@ -6,28 +6,22 @@ set -eo pipefail
 # Display commands being run
 set -x
 
-# Only run the linter on go1.11, since it needs type aliases (and we only care
+# Only run the linter on go1.12, since it needs type aliases (and we only care
 # about its output once).
-# TODO(deklerk) We should pass an environment variable from kokoro to decide
-# this logic instead.
-if [[ `go version` != *"go1.11"* ]]; then
+if [[ `go version` != *"go1.12"* ]]; then
     exit 0
 fi
 
-pwd
+export GO111MODULE=on
 
 # Fail if a dependency was added without the necessary go.mod/go.sum change
 # being part of the commit.
-GO111MODULE=on go mod tidy
+go mod tidy
 git diff go.mod | tee /dev/stderr | (! read)
```

Comment:
Is there any reason we're doing this twice?

Reply:
Assuming you meant the multiple GO111MODULE=on statements - fixed!